ompi/mca/mtl/portals4/mtl_portals4_component.c

```diff
@@ -50,8 +53,8 @@ mca_mtl_base_component_2_0_0_t mca_mtl_portals4_component = {
         OMPI_MINOR_VERSION,  /* MCA component minor version */
         OMPI_RELEASE_VERSION,  /* MCA component release version */
         ompi_mtl_portals4_component_open,  /* component open */
+        ompi_mtl_portals4_component_query,  /* component close */
```

Comment:
These assignments should be done with C99 syntax to avoid mis-ordering problems like this.